docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable 
+[restricting the three types of subpages or parent pages](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#parent-page-subpage-type-rules) 
+a particular page model can have. 
+
+For example, in our [interactive regulations page models](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+we have a `RegulationLandingPage` that can be created anywhere in the page tree. 
+`RegulationLandingPage`, however, can only have two types of pages created within it: 
+`RegulationPage` and `RegulationSearchPage`. 
+This parent/child relationship is expressed by setting `subpage_types` on `RegulationLandingPage`
+and `parent_page_types` on `RegulationPage` and `RegulationSearchPage` 
+to either the model classes themselves or to a model name in the form `app_label.ModelName`:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    subpage_types = ['regulations3k.RegulationPage', 'RegulationsSearchPage']
+
+
+class RegulationsSearchPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+
+
+class RegulationPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+```
+
+!!! note 
+    Note: we prevent child pages from being added to `RegulationPage` and `RegulationSearchPage` 
+    by setting `subpage_types` to an empty list.
+
+## Template rendering
+
+Sometimes new Wagtail page types will need to make customizations to their base template 
+[when rendering the page](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#template-rendering).
+This is done by overriding the `template` attribute on the page model.
+
+For example, the [interactive regulations landing page](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+includes a customized list of recently issued notices that gets loaded dynamically from the Federal Register. 
+To do this it provides its own template that inherits from our base templates 
+and overrides the `content_sidebar` block to include a seperate `recent_notices` template:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    template = 'regulations3k/landing-page.html'
+```
+
+And [`regulations3k/landing-page.html`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/jinja2/regulations3k/landing-page.html#L5):
+
+```jinja2
```

Comment:
Unfortunately, `jinja2` is not a supported syntax.

The GH suggestion feature doesn't work in this situation, but I suggest just removing it and not replacing it with anything else.

Reply:
Eh, what if it is supported at some future date? Just because it's not useful in code rendering doesn't mean it's not necessarily useless in the doc (I read Markdown for fun sometimes :P ).

Reply:
Well, it's _counterproductive_ to the rendering if there is actual HTML in the snippet, but since this case is exclusively Jinja tags, I suppose that's fine.